packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -43,8 +44,8 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     split: false,
   };
 
-  private _buttonElement: HTMLElement;
-  private _splitButtonContainer: HTMLElement;
+  private _buttonElement = createRef<HTMLElement>();
```

Comment:
yes! see, there is no reason to type the refs. It should be implied.

Reply:
True, we should discuss this one though https://github.com/palantir/tslint/issues/711 as i'm running into it when enabling `typedef`. (In the case of `typedef` it would require it ;))